### PR TITLE
19.1.16 trunk 1

### PR DESCRIPTION
**Pull request by amita-myfiziq**
Ticket: https://github.com/MyFiziqApp/proj-biomorphik/issues/1

### CODE REVIEW

File: MyFiziqTurnkey/Assets/myfiziq-turnkey.css [hunk 1]
```diff
@@ -77,6 +83,26 @@
...
+.myq-tk-input-view-restricted-access-title {
+    font: systemBold 22;
+    titleColor: @myqtkBlackColor;
+    numberOfLines: 0;
+    lineBreakMode: wordWrapping;
+    textAlignment: center;
+}
+
+.myq-tk-input-view-restricted-access-sub-title {
+    @extends .myq-tk-input-view-restricted-access-title;
```
Comment (phillipc-myfiziq) [inline MyFiziqTurnkey/Assets/myfiziq-turnkey.css:103]: Please change to `@extend` (i.e. drop the 's'), as InterfaCSS is finicky about this.

File: MyFiziqTurnkey/Assets/myfiziq-turnkey.css [hunk 2]
```diff
@@ -77,6 +83,26 @@
   backgroundColor: @myqtkBackgroundColor;
 }
 
+.myq-tk-input-view-restricted-access {
+    backgroundColor: @myqtkBackgroundColor;
```
Comment (phillipc-myfiziq) [inline MyFiziqTurnkey/Assets/myfiziq-turnkey.css:87]: Please mark style class with comment to indicate it's designation, e.g. `/* UIView */`

File: MyFiziqTurnkey/Classes/Private/MYQTKNew.m [hunk 3]
```diff
@@ -20,12 +20,17 @@
 #import <MyFiziqSDKInputView/MyFiziqInputView.h>
 #import <PureLayout/PureLayout.h>
 #import "MYQTKSubViewOnboarding.h"
+#import "MyFiziqTurnKey.h"
 
 // NOTE: Create the layout using PureLayout, not storyboards
 
-@interface MYQTKNew ()<MyFiziqInputViewDelegate, MyFiziqInputViewDataSource, MYQTKSubViewOnboardingDelegate>
+@interface MYQTKNew ()<MyFiziqInputViewDelegate, MyFiziqInputViewDataSource, MYQTKSubViewOnboardingDelegate, MyFiziqTurnkeyDatasourceDelegate>
```
Comment (phillipc-myfiziq) [inline MyFiziqTurnkey/Classes/Private/MYQTKNew.m:27]: `MYQTKNew` cannot be a Data source delegate. Only an instance implemented by the app can be a `MyFiziqTurnkeyDatasourceDelegate` type.

**Review by phillipc-myfiziq — CHANGES_REQUESTED** [review]
See comments

File: MyFiziqTurnkey/Classes/Private/MYQTKNew.m (hunk 3 above)
Comment (amita-myfiziq) [reply MyFiziqTurnkey/Classes/Private/MYQTKNew.m:27]: Yea i agree, its a copy paste error.

File: MyFiziqTurnkey/Assets/myfiziq-turnkey.css [hunk 4]
```diff
@@ -30,6 +30,12 @@
 @myqtkNavBarImageSizeForSmallState: 32.0;
 @myqtkNavBarHeightSmallState: 44.0;
 @myqtknavBarHeightLargeState: 96.5;
+@myqInputNoAccessViewTitleViewTop: 120;
+@myqInputNoAccessViewTitleViewLeft: 30;
+@myqInputNoAccessViewTitleViewRight: 30;
+@myqInputNoAccessViewTitleHeight: 100;
+@myqInputNoAccessViewSubTitleHeight: 90;
+@myqInputNoAccessViewSubTitleTop: 10;
```
Comment (phillipc-myfiziq) [inline MyFiziqTurnkey/Assets/myfiziq-turnkey.css:38]: As this is not an override to an Input SDK style variable, the prefix should be `myqtk`, not `myqInput`

**Review by phillipc-myfiziq — CHANGES_REQUESTED** [review]
just one minor comment to look at

**Review by phillipc-myfiziq — APPROVED** [review]

**Review by phillipc-myfiziq — APPROVED** [review]

**Pull request merged**